### PR TITLE
Refactoring the User components/redux and Tests

### LINKED ISSUE #100: Refactor User Redux components

### LINKED ISSUE #102: Refactor Login/Logout/Register/Forgot
All into one directory... and then use an `index.js` file

### PR DESCRIPTION
**Pull request by Mortuie**
Closes #100 & Closes #102

### CODE REVIEW

File: frontend/src/User/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Login from './Login';
+import Register from './Register';
+import Forgot from './Forgot';
+import Logout from './Logout';
+import userReducer from './reducer';
+
+export { Login, Logout, Register, Forgot , userReducer};
```
Comment (nikrb) [inline frontend/src/User/index.js]: needs newline at end of file

**Review by nikrb — APPROVED** [review]
looks good.

**Comment by Mortuie** [comment]
@nikrb have I added a new line at the end of the file? Also Do we need two sets of actions or can I put them into one file?

**Comment by nikrb** [comment]
files should have a blank line at the end ... it's a git thing.
I would suggest two sets of actions, unless they are related in which case put them in one file.

**Comment by Mortuie** [comment]
@nikrb all the actions are related to logging in and producing errors etc with respect to this...

**Comment by nikrb** [comment]
If they're all related to login, then one file sounds good

**Comment by Mortuie** [comment]
@nikrb will do this later then!

**Review by vivekimsit — APPROVED** [review]

**Comment by Mortuie** [comment]
@nikrb @vivekimsit put all the actions into one file please review again. Also broke a test.

**Comment by nikrb** [comment]
lol @ cross-fetch nonsense

**Comment by Mortuie** [comment]
also see 
image

**Comment by Mortuie** [comment]
@nikrb @vivekimsit commented out pre and post pull/push stufff and resized register box aswell. Final comments and then merge this beast. ty

**Review by vivekimsit — APPROVED** [review]

**Pull request merged**